.github/workflows/integration-tests.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       working-directory: ./openverse-api
       run: |
         pip install --upgrade pipenv
```

Comment:
And if we stop using `pipenv`, we can also drop this.

Reply:
Okay